src/diffusers/models/attention.py

```diff
@@ -515,3 +521,268 @@ def forward(self, x, emb):
         x = F.group_norm(x, self.num_groups, eps=self.eps)
         x = x * (1 + scale) + shift
         return x
+
+
+# WIP Tune a Video classes
+
+
+@dataclass
+class Transformer3DModelOutput(BaseOutput):
+    sample: torch.FloatTensor
+
+
+class Transformer3DModel(ModelMixin, ConfigMixin):
```

Comment:
I think it would be better to give this model its own module as we have for `Transformer2DModel`.

Reply:
Ohh sure. Let me take a look at the `Transformer2DModel` module.